### ISSUE TITLE
related-words writes over previous calls to related-words

### ISSUE THREAD

**Issue by erg** [factor/factor#3047]
```
{ 3if 3when 3unless } related-words
{ 1if 2if 3if } related-words
```

The second one wins:

<img width="307" alt="Screenshot 2024-06-22 at 17 35 27" src="https://github.com/factor/factor/assets/27430/50387b19-75d9-4acd-bbbf-9d9899b592de">

It would be better if they added to the related words set or even if you could relate words across multiple dimensions:

```
{
    { 3if 3when 3unless }
    { 1if 2if 3if }
} related-words
```

**Comment by mrjbq7** [comment]
No because adding wouldn’t make reload work right.

**Comment by erg** [comment]
This is why we have syntax forms? :)

```
RELATED-WORDS: 3if 3when 3unless ;
```

**Comment by mrjbq7** [comment]
I don’t think that solves anything On Jun 22, 2024, at 3:38 PM, Doug Coleman ***@***.***> wrote:﻿
This is why we have syntax forms? :)
RELATED-WORDS: 3if 3when 3unless ;

**Comment by erg** [comment]
You also use the redefinition tracking and it can work out.

**Comment by mrjbq7** [comment]
Only if you restrict it to same vocab related-words I think. Otherwise it wouldn’t be cleanup-able without extra work. On Jun 22, 2024, at 3:53 PM, Doug Coleman ***@***.***> wrote:﻿
You also use the redefinition tracking and it can work out.
